ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -312,6 +276,7 @@ export class DataMapperHostComponent extends FlowPage implements OnInit {
     this.initializationService.resetConfig();
     this.cfg = this.initializationService.cfg;
 
+    // TODO: Move hardcoded values to data service
```

Comment:
Lets create an issue for this item

Reply:
I'm putting together an EPIC that encompasses a handful of issues around forcing the Angular application architecture model (façades, error logging, exception handling, single responsiiblity principle, path aliases, schematics) into Syndesis, along with some other (hopefully minor) code refactorings necessary to finally embrace the Angular Styleguide. 

One of those issues cater this matter already :)